### ISSUE TITLE
Automatic update is broken

### ISSUE THREAD

**Issue by traversaro** [conda-forge/gz-sim-feedstock#65]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

No idea why.

### Installed packages

```shell
.
```

### Environment info

```shell
.
```

**Comment by traversaro** [comment]
Triggering an update in gave us an error in https://github.com/conda-forge/gz-sensors-feedstock/actions/runs/10538400204/job/29200871549 :

~~~
2024-08-24 12:17:28,030 INFO     webservices_dispatch_action.version_updater || getting latest version
2024-08-24 12:17:41,638 INFO     webservices_dispatch_action.version_updater || curr version|latest version: 8.1.0|8.2.0
2024-08-24 12:17:41,658 INFO     conda_forge_tick.update_recipe.version || selector: None
2024-08-24 12:17:41,663 INFO     conda_forge_tick.update_recipe.version || jinja2 context: {'component_name': 'sensors', 'version': '8.2.0'}
2024-08-24 12:17:41,663 INFO     conda_forge_tick.update_recipe.version || evaluated jinja2 vars: {'repo_name': 'gz-sensors', 'major_version': '8.2.0', 'version_package': '', 'name': 'gz-sensors8.2.0', 'component_version': 'sensors8.2.0', 'cxx_name': 'libgz-sensors8.2.0'}
2024-08-24 12:17:41,663 INFO     conda_forge_tick.update_recipe.version || updated jinja2 context: {'component_name': 'sensors',
 'component_version': 'sensors8.2.0',
 'cxx_name': 'libgz-sensors8.2.0',
 'major_version': '8.2.0',
 'name': 'gz-sensors8.2.0',
 'repo_name': 'gz-sensors',
 'version': '8.2.0',
 'version_package': ''}
2024-08-24 12:17:41,663 INFO     conda_forge_tick.update_recipe.version || url key: url
2024-08-24 12:17:41,663 INFO     conda_forge_tick.update_recipe.version || hash key: sha256
2024-08-24 12:17:41,663 INFO     conda_forge_tick.update_recipe.version || hashing URL template: [https://github.com/gazebosim/{{](https://github.com/gazebosim/%7B%7B) repo_name }}/archive/{{ repo_name }}{{ version }}.tar.gz
2024-08-24 12:17:41,664 INFO     conda_forge_tick.update_recipe.version || rendered URL: https://github.com/gazebosim/gz-sensors/archive/gz-sensors8.2.0.tar.gz
2024-08-24 12:17:43,910 INFO     conda_forge_tick.update_recipe.version || new URL hash: None
2024-08-24 12:17:43,911 INFO     conda_forge_tick.update_recipe.version || updated|errors: False|{"could not hash URL template '[https://github.com/gazebosim/{{](https://github.com/gazebosim/%7B%7B) repo_name }}/archive/{{ repo_name }}{{ version }}.tar.gz'"}
2024-08-24 12:17:43,911 CRITICAL conda_forge_tick.update_recipe.version || {"could not hash URL template '[https://github.com/gazebosim/{{](https://github.com/gazebosim/%7B%7B) repo_name }}/archive/{{ repo_name }}{{ version }}.tar.gz'"}
2024-08-24 12:17:43,912 CRITICAL conda_forge_tick.update_recipe.version || Recipe did not change in version migration!
2024-08-24 12:17:43,912 CRITICAL webservices_dispatch_action.version_updater || errors when updating the recipe: {"could not hash URL template '[https://github.com/gazebosim/{{](https://github.com/gazebosim/%7B%7B) repo_name }}/archive/{{ repo_name }}{{ version }}.tar.gz'"}
2024-08-24 12:17:43,912 ERROR    webservices_dispatch_action.version_updater || error while updating the recipe!
Traceback (most recent call last):
  File "/webservices_dispatch_action/webservices_dispatch_action/version_updater.py", line 97, in update_version
    raise RuntimeError("Error updating the recipe!")
RuntimeError: Error updating the recipe!
INFO:webservices_dispatch_action.utils:token can change workflows: False
INFO:webservices_dispatch_action.utils:pushing and commenting: branch|owner|repo = conda_forge_admin_37|conda-forge-admin|gz-sensors-feedstock
Traceback (most recent call last):
  File "/opt/conda/bin/run-webservices-dispatch-action", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/webservices_dispatch_action/webservices_dispatch_action/__main__.py", line 175, in main
    raise RuntimeError(
RuntimeError: Updating version failed! error in push|version update: False|True
~~~

**Comment by traversaro** [comment]
No idea why, but apparently to fix this we need to align the recipe style with gz-math: https://github.com/conda-forge/gz-sensors-feedstock/commit/5bbc306bf23c84d65cd1e81d3a930614081e2990 .

**Comment by traversaro** [comment]
We need to review and align all repos in https://github.com/orgs/conda-forge/repositories?q=gz- .

**Comment by Tobias-Fischer** [comment]
It looks like all of these have been fixed @traversaro?

**Comment by traversaro** [comment]
To be honest, I do not know. We should check if a autotick-bot PR was opened in one of this repos after the cleanup PRs.

**Comment by Tobias-Fischer** [comment]
Nope .. there is gz-physics 7.3.0 but there was no autotick-bot :(

**Comment by traversaro** [comment]
> Nope .. there is gz-physics 7.3.0 but there was no autotick-bot :(

Yes, interesting. There is no error in versions error in conda-forge status, and if you check the bot-versions output it outputs (see https://github.com/regro/cf-scripts/actions/runs/11007210867/job/30562973811):

~~~
2024-09-24T05:29:51.8730580Z 2024-09-24 05:29:51,872 INFO     conda_forge_tick.update_upstream_versions || itr  2236 - eta  2226s: gz-physics - 7.2.0 -> 7.3.0
~~~

so the correct version is detected, but somehow the PR is not opened.

**Comment by traversaro** [comment]
Opened an upstream issue at https://github.com/regro/cf-scripts/issues/3010 .

**Comment by traversaro** [comment]
> The only missing repo is gz-sim, that will be fixed once #64 is merged.

Actually it is now fixed, see https://github.com/conda-forge/gz-sim-feedstock/pull/67 . Let's close the issue.
